### DISCUSSION TITLE
potatophant bought potatophant.net! YEEEEEESSSSS!!!1!!!!11!!

### DISCUSSION THREAD

**Discussion by brourbeinsus** [CluckCluckChicken/Magnifier#53] [big chungus]
SUS DEACTIVATED!!!!111!!!!11!!!1!1!!

**Comment by JackyBorderCollie** [comment]
YEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEESSSSSSSSSSSSSSSSSSSSS111!111!!!11!!!11111!!!11!!!

**Comment by davwat** [comment]
hang on do i have to sell it in order to close this issue??//?//?////???

**Reply by brourbeinsus** [reply]
nOoOoOoOoOoOoOoOoOoOoOo

**Comment by davwat** [comment]
eauf its not even an issue

**Comment by FunctionalMetatable** [comment]
mAkE a NuXt sItE aT potatophant.net plz

**Comment by JackyBorderCollie** [comment]
OMG this sussy pig is preventing from accessing the site!!1!1!!!
![](https://u.cubeupload.com/JackyBorderCollie/Sussypig.png)

**Reply by brourbeinsus** [reply]
Noooooooooooooooo

**Reply by FunctionalMetatable** [reply]
:suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: :suspect: v
